SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -14,30 +15,39 @@ public final class SwiftGenStoryboardEnumBuilder {
         
         class ParserDelegate : NSObject, NSXMLParserDelegate {
             var scenes = [Scene]()
+            var segues = [Scene]()
             var inScene = false
             var readyForFirstObject = false
+            var readyForConnections = false
             
             @objc func parser(parser: NSXMLParser, didStartElement elementName: String,
-                namespaceURI: String?, qualifiedName qName: String?,
-                attributes attributeDict: [String : String])
+              namespaceURI: String?, qualifiedName qName: String?,
+              attributes attributeDict: [String : String])
             {
-                
-                switch elementName {
-                case "scene":
-                    inScene = true
-                case "objects" where inScene:
-                    readyForFirstObject = true
-                case _ where readyForFirstObject:
-                    if let storyboardID = attributeDict["storyboardIdentifier"] {
-                        let customClass = attributeDict["customClass"]
-                        scenes.append(Scene(storyboardID, customClass))
-                    }
-                    readyForFirstObject = false
-                default:
-                    break
+              
+              switch elementName {
+              case "scene":
+                inScene = true
+              case "objects" where inScene:
+                readyForFirstObject = true
+              case _ where readyForFirstObject:
+                if let storyboardID = attributeDict["storyboardIdentifier"] {
+                  let customClass = attributeDict["customClass"]
+                  scenes.append(Scene(storyboardID, customClass))
                 }
+                readyForFirstObject = false
+              case "connections":
+                readyForConnections = true
+              case _ where readyForConnections:
```

Comment:
Just checked on another project of mine and here's a `Storyboard` scene with outlets
```xml
<connections>
    <outlet property="hairlineView" destination="VRg-OY-4IC" id="PuB-XN-okm"/>
    <outlet property="portTextField" destination="6b3-TB-J53" id="Ks7-gi-ZD7"/>
    <outlet property="serverTextField" destination="Lfy-mJ-Msa" id="apH-fb-2cT"/>
    <segue destination="Z5I-uz-FR0" kind="show" identifier="ShowClientsSegue" id="cml-0d-ha0"/>
</connections>
```